src/app/courses/[course]/lectures/[lectureId]/exercises/[exerciseId]/page.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { api } from "@/trpc/server";
+import ExerciseInfoBox from "@/components/codeSpace/exerciseInfoBox";
+import ResizableDemo from "@/components/codeSpace/Resizable";
+
+
+export default async function page({
+  params,
+}: {
+  params: { exerciseId: string };
+}) {
+  const exercise = await api.exercise.getExercise({
```

Comment:
i honestly, dont know, because the reziable component has three other components "exerciesInfoBox", "combobox" and "testsuite" and i kinda dont know what this part actually does, so... prop need help with it 😁